src/index.js

```diff
@@ -39,7 +40,10 @@ class FtpServer extends EventEmitter {
       _.get(this, 'options.pasv_min'),
       _.get(this, 'options.pasv_max'));
 
+	this.timeout = isNaN(Number(this.options.timeout)) ? 0 : Number(this.options.timeout);
```

Comment:
I liked having a scoped variable to get the timeout. But set it on the options timeout, not a timeout directly on the instance.

```suggestion
      const timeout = Number(this.options.timeout);
      this.options.timeout = isNaN(timeout) ? 0 : Number(timeout);
```